src/styles/card.scss

```diff
@@ -0,0 +1,90 @@
+.card {
+  width: $card-width;
+  border: $card-border;
+  border-radius: 5px;
+  box-sizing: border-box;
+
+  &__image {
+    background-image: url(../images/imac.jpeg);
+    height: 134px;
+    width: 160px;
+    background-size: 160px 134px;
+    margin: 32px 19px 40px;
+  }
+
+  &__title {
+    font-weight: 500;
+    color: $color-main;
+    font-size: 12px;
+    line-height: 18px;
+    margin: 0 16px 4px;
+  }
+
+  &__code {
+    font-weight: 400;
+    color: $color-secondary;
+    font-size: 10px;
+    line-height: 14px;
+    margin: $margin;
+  }
+
+  &__stars {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-left: $margin-left;
```

Comment:
The variable `$margin-left` is used but not defined. Ensure it is defined in the `variables.scss` file or imported correctly.